src/app/Main/Body/InfoPage/InfoPage.component.tsx

```diff
@@ -52,11 +70,20 @@ export function InfoPageComponent(props: {
     return (
         <div css={infoPageStyle}>
             <p css={headingStyle}>{t("Title")}</p>
-            <InfoPageNavButtonsComponent
-                backgroundButtonLabel={backgroundChapterHeading}
-                filtersButtonLabel={filtersChapterHeading}
-                methodsButtonLabel={methodsChapterHeading}
-            />
+            <section css={navButtonDisplayStyle}>

```

Comment:
i wonder why you used a  section element here?

Reply:
Because that's how it was done in the mibi portal. haha I know - that's not a good answer. Would a div be enough?

Reply:
yes, if you want to use sections you have to read the docs for that feature and refactor the whole InfoPage html accordingly.

Reply:
So defining semantic elements like section is for algorithms scanning your web page, so they know how the document is formed (e.g screen-readers). But that's another story and should be done in a separate ticket if the feature is requested.

Reply:
And yes, be careful about adapting code from the mibi-portal, it needs heavy refactoring on some parts of the app, especially the component and folder structure.

Reply:
Done. Commit "Remove section & unused background button"